tests/lib/cli-engine.js

```diff
@@ -1457,12 +1535,13 @@ describe("CLIEngine", function() {
 
                 engine.executeOnFiles([file]);
 
-                assert.isTrue(fs.existsSync(path.resolve(".eslintcache")), "the cache for eslint was created");
+                assert.isTrue(fs.existsSync(cacheFile), "the cache for eslint was created");
 
                 engine = new CLIEngine({
                     useEslintrc: false,
                     // specifying cache true the cache will be created
                     cache: false,
+                    cacheFile: cacheFile,
```

Comment:
It would be best not to make this change here, since it's unrelated to the ignore file work. We like to keep pull requests as focused as possible in case there's a problem later.